### ISSUE TITLE
SOLVED Method '_FilesCollectionStart_fotos' not found [404]

### ISSUE THREAD

**Issue by felipesmendes** [VeliovGroup/Meteor-Files#296]
I'm using Meteor 1.4.2.3 with React without Blaze, and trying to upload a photo using Meteor-Files, when I call insert from FilesCollection I'm recieving this error:

![image](https://cloud.githubusercontent.com/assets/992965/20560182/18106114-b15f-11e6-8e45-bde86ef58c4a.png)

Check the code:

```
var upload = Fotos.insert({
            file: e.target.files[0],
            streams: 'dynamic',
            chunkSize: 'dynamic'
          }, false);
```

My Collection:
```

  import {
    Meteor
  }
  from 'meteor/meteor';
  import {
    FilesCollection
  }
  from 'meteor/ostrio:files';
  SimpleSchema.debug = true;
  Fotos = new FilesCollection({
    collectionName: 'fotos',
    debug: true,
    storagePath: "/fotos/uploads",
    allowClientCode: true, // Disallow remove files from Client
    onBeforeUpload: function(file) {
      return true;
    },
  });
  export default Fotos;
```

**Comment by felipesmendes** [comment]
Can you close... fixed using #110

**Comment by samybob1** [comment]
Thanks @felipesmendes, I had the same problem, [this comment is the answer](https://github.com/VeliovGroup/Meteor-Files/issues/110#issuecomment-232883827).
